### ISSUE TITLE
CI fails due to missing `apt-get update`

### ISSUE THREAD

**Issue by kkarbowiak** [VROOM-Project/vroom#863]

**Comment by kkarbowiak** [comment]
It seems that the CI fails due to other reasons, but I think adding `sudo apt-get update` in CI jobs is still a good idea.

**Comment by jcoupey** [comment]
Yes, during the dependency installation, the `Could not connect to azure.archive.ubuntu.com:80 (52.154.174.208), connection timed out` part for https://github.com/VROOM-Project/vroom/actions/runs/3909516920/jobs/6680678212 sounds like we're not at fault here.

**Comment by jcoupey** [comment]
Done in #864
